modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/source.jsp

```diff
@@ -96,23 +96,23 @@ List<Map<String, Object>> classTypesList = new ArrayList<>();
 
 		Long[] assetSelectedClassTypeIds = assetPublisherWebHelper.getClassTypeIds(portletPreferences, className, classTypes);
 
-		// Left list
+		// Right list
 
-		List<KeyValuePair> subtypesLeftList = new ArrayList<KeyValuePair>();
+		List<KeyValuePair> subtypesRightList = new ArrayList<KeyValuePair>();
```

Comment:
This Right list looks like it doesn't need to be before the Left list to me. Is that correct? Can we move it down below the left list creation, just above the <liferay-ui:input-move-boxes declaration?